himin/day2/수박수박수박수박수?.js

```diff
@@ -0,0 +1,3 @@
+function solution(n) {
+    return "수박".repeat(Math.floor (n / 2)) + (n % 2 == 1 ? "수" : "")
```

Comment:
```suggestion
    return "수박".repeat(Math.floor(n / 2)) + (n % 2 === 1 ? "수" : "")
```

동등연산자(==) 보다는 일치연산자(===) 사용을 추천드립니다.
참고 : https://github.com/modern-agile-team/9term-assignment/pull/12#discussion_r1869539229

Reply:
https://github.com/modern-agile-team/9term-assignment/commit/498f1d69f7451b6242ee8bdd456669fbd67ced78

@ssi02014 일치연산자 사용하여 커밋하였습니다!